util/src/main/java/io/grpc/util/AdvancedTlsX509TrustManager.java

```diff
@@ -211,12 +218,15 @@ private void checkTrusted(X509Certificate[] chain, String authType, SSLEngine ss
 
   /**
    * Schedules a {@code ScheduledExecutorService} to read trust certificates from a local file path
-   * periodically, and update the cached trust certs if there is an update.
+   * periodically, and update the cached trust certs if there is an update. Please make sure to
```

Comment:
micro-nit: s/and update the/and updates the

Reply:
2 instances fixed, thank you!